docs/source/architecture/rest_api.rst

```diff
@@ -2,313 +2,47 @@
 REST API
 ********
 
-*Hyperledger Sawtooth* provides a pragmatic RESTish API for clients to interact
-with a validator using common HTTP/JSON standards. It is an entirely separate
-process, which once running, allows transactions to be submitted and blocks to
-be read with a common language-neutral interface. As the validator is redesigned
-and improved the REST API will grow with it, providing a consistent interface
-that meets the needs of application developers into the future.
-
-With that focus on application developers, the REST API treats the validator
-mostly as a
-black box, submitting transactions and fetching the results. It is not the tool
-for all validator communication. For example, it is not used by Transaction
-Processors to communicate with a validator, or by one validator to talk to other
-validators. For these and similar use cases, there is a more efficient and
-robust, if somewhat more complicated, ZMQ/Protobuf interface.
-
-
-Open API Specification
-======================
-
-The REST API is comprehensively documented using the
-`OpenAPI specification <http://swagger.io/specification/>`_ (fka Swagger),
-formatted as a YAML file. This documentation provides a single source of truth
-that completely documents every implemented aspect of the API, is both human and
-machine readable, and can be compiled by a variety of toolsets (including for
-this Sphinx-based document). The spec file itself can be found
-`here <https://github.com/hyperledger/sawtooth-core/blob/master/rest_api/openapi.yaml>`_.
-
-
-HTTP Status Codes
+Hyperledger Sawtooth provides a REST API (see the :doc:`../rest_api`) that
+allows clients to interact with a validator using common HTTP/JSON standards.
+It is a pragmatic RESTish API that provides a language-neutral interface for
+submitting transactions and reading blocks.
+
+The Sawtooth REST API is intended only as a simple interface for client use. The
+validator and transaction processors use a ZMQ/Protobuf interface [LINK]
+that is more efficient and robust, as well as slightly more complicated.
+Note that clients also have the option of using the ZMQ/Protobuf interface to
+communicate with the validator.
+
+Note that the REST API is a lightweight layer on top of Sawtooth's internal ZMQ
```

Comment:
You start two sentences in a row with "Note that".

Reply:
:-(  I'll fix that.